.github/workflows/unit-test.yml

```diff
@@ -196,10 +193,10 @@ jobs:
         run: python -m pytest ${{ inputs.pytest_test_directory }} ${{ steps.pytest_args.outputs.args }} ${{ env.PYTEST_MARKER }} --disable-opmath=${{ inputs.disable_new_opmath }}
 
       - name: Upload Durations file as artifact
-        if: ${{ inputs.pytest_store_durations == true }}
+        if: ${{ inputs.pipeline_mode == 'update-durations' }}
         uses: actions/upload-artifact@v4
         with:
-          name: ${{ inputs.job_name }}-durations.json
+          name: durations-${{ inputs.job_name }}
           path: ${{ inputs.pytest_durations_file_path }}
           include-hidden-files: true
```

Comment:
Only upload durations as artifacts if using the `update-durations` pipeline mode. Also changed the artifact name to start with `durations-` to be easier to search for when downloading the artifacts